bedrock/firefox/views.py

```diff
@@ -118,16 +115,39 @@ def stub_attribution_code(request):
         codes['source'] = 'www.mozilla.org'
         codes['medium'] = '(none)'
 
-    codes['timestamp'] = str(int(time()))
+    response = HttpResponseJSON(sign_attribution_codes(codes))
+    patch_response_headers(response, 300)  # 5 min
+    return response
+
+
+def get_attrribution_code(codes):
+    """
+    Take the attribution codes and return the URL encoded string
+    respecting max length.
+    """
     code = '&'.join('='.join(attr) for attr in codes.items())
```

Comment:
Just clarifying that we're checking the length of `code` prior to base64 encoding?

Reply:
Right. This is what the new changes to the attribution service do. There's a 200 char limit on the decoded data.